app/Http/Requests/LoginRequest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class LoginRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'email' => ['required', 'email', 'max:255',],
+            'password' => ['required', 'string', 'min:8', 'max:20',],
```

Comment:
```suggestion
            'email' => ['required', 'email', 'max:320'],
            'password' => ['required', 'string', 'min:8', 'max:20'],
```
mảng trên 1 dòng ko có dấu , ở phần tử cuối nha